packages/components/PersonaCoin/src/PersonaCoin.helpers.ts

```diff
@@ -52,34 +53,34 @@ const sizeTable: {[P in PersonaSize]: PersonaSizeConfig} = {
   size56: { physicalSize: 56, iconSize: 16, iconStrokeWidth: 3, initialsSize: 18 },
   size72: { physicalSize: 72, iconSize: 20, iconStrokeWidth: 3, initialsSize: 20 },
   size100: { physicalSize: 100, iconSize: 28, iconStrokeWidth: 4, initialsSize: 36 },
-  size120: { physicalSize: 120, iconSize: 32, iconStrokeWidth: 4, initialsSize: 40 }
+  size120: { physicalSize: 120, iconSize: 32, iconStrokeWidth: 4, initialsSize: 40 },
 };
 
 export function getSizeConfig(size: PersonaSize): PersonaSizeConfig {
   return sizeTable[size];
 }
 
 const colorTable: { [P in PersonaCoinColor]: string } = {
-  lightBlue: '#4F6BED',
-  blue: '#0078D4',
-  darkBlue: '#004E8C',
-  teal: '#038387',
-  green: '#498205',
-  darkGreen: '#0B6A0B',
-  lightPink: '#C239B3',
-  pink: '#E3008C',
-  magenta: '#881798',
-  purple: '#5C2E91',
-  orange: '#CA5010',
-  lightRed: '#D13438',
-  darkRed: '#A4262C',
-  violet: '#8764B8',
-  gold: '#986F0B',
-  burgundy: '#750B1C',
-  warmGray: '#7A7574',
-  cyan: '#005B70',
-  rust: '#8E562E',
-  coolGray: '#69797E'
+  lightBlue: globalTokens.color.cornflower.primary,
+  blue: globalTokens.color.blue.primary,
+  darkBlue: globalTokens.color.royalBlue.primary,
+  teal: globalTokens.color.teal.primary,
+  green: globalTokens.color.forest.primary,
+  darkGreen: globalTokens.color.darkGreen.primary,
+  lightPink: globalTokens.color.berry.primary,
+  pink: globalTokens.color.hotPink.primary,
+  magenta: globalTokens.color.grape.primary,
+  purple: globalTokens.color.purple.primary,
+  orange: globalTokens.color.pumpkin.primary,
+  lightRed: globalTokens.color.red.primary,
+  darkRed: globalTokens.color.burgundy.primary,
+  violet: globalTokens.color.orchid.primary,
+  gold: globalTokens.color.brass.primary,
+  burgundy: globalTokens.color.darkRed.primary,
```

Comment:
could we also just.. update the key names to match?

Reply:
Still trying to get a handle on what we're okay with breaking vs. not. Long run it makes sense update these so that the names all align, talking with @FalseLobster on whether this is an okay change now.

Reply:
Introduced a new prop, `coinColorFluent`, and marked the old prop as deprecated. Updated the test page to use the new prop.